openfoodfoundation/openfoodnetwork
# Store link to open backorder

#### :information_source: Funded Feature. Please track ALL ASSOCIATED WORK under the associated tracking code `#11678 DFC Orders`


#### What? Why?

- Closes #12952 



A new scenario is be possible now:

- Distributor has two order cycles open with products from the same DFC supplier.
- Previously, we would create only one backorder.
- The backorder got finalised at the end of the first order cycle.
- Now we create a backorder for each order cycle.
- The backorders are amended independently for each order cycle.
- The backorders get finalised at the end of their respective order cycle.


#### What should we test?


- Testing for @RaggedStaff:
- Open new order cycle with DFC products.
- Place orders which trigger a backorder.
- Cancel one order -> backorder should get adjusted as before.
- At OC end, the backorder should get finalised like before.


#### Release notes



Changelog Category (reviewers may add a label for the release notes):




The title of the pull request will be included in the release notes.


#### Dependencies




#### Documentation updates
